### PR TITLE
Minor changes in comments

### PR DESCRIPTION
**Pull request by PyAgni**

### CODE REVIEW

File: src/commoncode/codec.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def to_base_n(num, base):
 def to_base85(num):
     """
     Convert `num` number to a string representing this number in base 85,
-    padded as needed.
+    padding is done as needed.
```
Comment (pombredanne) [inline src/commoncode/codec.py]: Are you sure this is a valuable comment update?

Comment (PyAgni) [reply src/commoncode/codec.py]: Now realizing that this isn't of much value. Reverting this to as it was.

**Comment by PyAgni** [comment]
> Thanks for the patch! No patch is too small... though can you check the guidelines in https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst > I would need a sign off on your commit.
> See also my feeback inline

Added the sign off on the commit. I amended the commit and force pushed it, but some errors occurred in the checks.

File: src/commoncode/codec.py (hunk 1 above)
Comment (PyAgni) [reply src/commoncode/codec.py]: > @PyAgni Thank you for the update. Do you mind to squash these two commits in a single one?

Done. Hope everything seems good now.

**Comment by pombredanne** [comment]
Thanks your commit comments looks not right now:
```
Made minor changes in comments

Update codec.py

Signed-off-by: Agni Bhattacharyya <bhattacharyya.agni@gmail.com>

Squashed the commits.
Signed-off-by: Agni Bhattacharyya <bhattacharyya.agni@gmail.com>
```

Instead it should be more like this:
```
Update to_base10() docstring in codec.py

Add proper markup to argument names.

Signed-off-by: Agni Bhattacharyya <bhattacharyya.agni@gmail.com>
```

Please read this https://github.com/nexB/aboutcode/wiki/Writing-good-commit-messages

**Comment by PyAgni** [comment]
Changed the commit comments. Please have a look.

**Comment by pombredanne** [comment]
Thanks!

**Pull request merged**